common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -59,7 +58,7 @@
                     // ID is actually pointing to the article, which is wrong. It would be nice to have the ID on the
                     // ContentCard, but we don't for now. Annoyingly this doesn't allow us to check for if we should
                     // play ads or if the video is expired, but as fronts change really often, this is a non-problem.
-                    item.mediaType.flatMap(_ => item.id)
+                    if (item.isMediaLink) item.id else None
```

Comment:
Same as before, the embed path is not added to the video (as it shouldn't be with article links).